apps/mull-ui/src/utilities.ts

```diff
@@ -16,3 +22,46 @@ export const formatDate = (
 
   return { year: date.getFullYear(), month, day: date.getDate(), time: timeString };
 };
+
+/**
+ * Utility method for logging in the user. Used in the login page and registration page
+ *
+ * @param login the login mutation
+ * @param loginInput the login input containing the variables for the mutation
+ * @param updateToast
+ * @param setAccessToken
+ * @param setUserId
+ * @param history
+ *
+ * @see RegisterPage
+ * @see LoginPage
+ */
+export const loginUser = async (
+  login,
+  loginInput: ILoginForm,
+  updateToast: (type: TypeOptions, message: string) => void,
+  setAccessToken: React.Dispatch<React.SetStateAction<string>>,
+  setUserId: React.Dispatch<React.SetStateAction<number>>,
+  history: History
+) => {
```

Comment:
I feel like this function has way too many parameters and I assume you pulled it out to account for the fact that you're using it in different places. IMO, I think that login logic should only belong to the login component.

Reply:
> I assume you pulled it out to account for the fact that you're using it in different places

You're right on the money. Since I'll no longer login the user from the register page, I'll be removing this method.

> IMO, I think that login logic should only belong to the login component.

I will be moving this logic back to the login page.